src/DependencyInjection/Configuration.php

```diff
@@ -39,6 +40,13 @@ public function getConfigTreeBuilder(): TreeBuilder
             ->children()
                 ->scalarNode('proxy_namespace')->defaultValue('MongoDBODMProxies')->end()
                 ->scalarNode('proxy_dir')->defaultValue('%kernel.cache_dir%/doctrine/odm/mongodb/Proxies')->end()
+                ->booleanNode('enable_lazy_ghost_objects')
```

Comment:
Same name used in the ORM: https://github.com/doctrine/DoctrineBundle/blob/2363c43d9815a11657e452625cd64172d5587486/src/DependencyInjection/Configuration.php#L512